src/Controller.php

```diff
@@ -113,6 +114,12 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
                     try {
                         return $this->getResultEncoder()->encode(call_user_func([&$this, $action], $request, $arguments), $request, $response);
                     } catch (Exception $e) {
+                        if ($this->logger instanceof LoggerInterface) {
+                            $this->logger->warning("Exception is caught with message {$e->getMessage()}", [
```

Comment:
Because exception is caught and handled and we can not do anything about it in current request, maybe it is better to log it as a warning. However, error is also ok. 

Sent from my iPhone

> On Jul 8, 2016, at 17:31, Ilija Studen <notifications@github.com> wrote:
> 
> In src/Controller.php:
> 
> > @@ -113,6 +114,12 @@ public function __invoke(ServerRequestInterface $request, ResponseInterface $res
> >                      try {
> >                          return $this->getResultEncoder()->encode(call_user_func([&$this, $action], $request, $arguments), $request, $response);
> >                      } catch (Exception $e) {
> > +                        if ($this->logger instanceof LoggerInterface) {
> > +                            $this->logger->warning("Exception is caught with message {$e->getMessage()}", [
> Warning or error?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>